backend/app/views/spree/admin/shared/_order_submenu.html.erb

```diff
@@ -1,61 +1,61 @@
       </li>
     <% end %>
 
     <% if checkout_steps.include?("address") %>
-      <li<%== ' class="active"' if current == 'Customer Details' %> data-hook='admin_order_tabs_customer_details'>
+      <li class="<%= "active" if (current == "Customer Details") %>" data-hook='admin_order_tabs_customer_details'>
         <% if can?(:update, @order) %>
-          <%= link_to_with_icon 'user', Spree.t(:customer_details), edit_admin_order_customer_url(@order) %>
+          <%= link_to_with_icon 'user', Spree.t(:customer), edit_admin_order_customer_url(@order) %>
         <% else %>
-          <%= link_to_with_icon 'user', Spree.t(:customer_details), admin_order_customer_url(@order) %>
+          <%= link_to_with_icon 'user', Spree.t(:customer), admin_order_customer_url(@order) %>
         <% end %>
       </li>
     <% end %>
 
-    <li<%== ' class="active"' if current == 'Shipments' %> data-hook='admin_order_tabs_order_details'>
+    <li class="<%= "active" if current == "Shipments" %>" data-hook='admin_order_tabs_order_details'>
       <%= link_to_with_icon 'edit', Spree::Shipment.model_name.human(count: :other), edit_admin_order_url(@order) %>
     </li>
 
     <% if can? :display, Spree::Adjustment %>
-      <li<%== ' class="active"' if current == 'Adjustments' %> data-hook='admin_order_tabs_adjustments'>
+      <li class="<%= "active" if current == "Adjustments" %>" data-hook='admin_order_tabs_adjustments'>
         <%= link_to_with_icon 'cogs', Spree::Adjustment.model_name.human(count: :other), admin_order_adjustments_url(@order) %>
       </li>
     <% end %>
 
     <% if can?(:display, Spree::Payment) %>
-      <li<%== ' class="active"' if current == 'Payments' %> data-hook='admin_order_tabs_payments'>
+      <li class="<%= "active" if current == "Payments" %>" data-hook='admin_order_tabs_payments'>
         <%= link_to_with_icon 'credit-card', Spree::Payment.model_name.human(count: :other), admin_order_payments_url(@order) %>
       </li>
     <% end %>
 
     <% if !@order.completed? && can?(:update, @order) %>
-      <li<%== ' class="active"' if current == 'Confirm' %> data-hook='admin_order_tabs_confirm'>
+      <li class="<%= "active" if current == "Confirm" %>" data-hook='admin_order_tabs_confirm'>
         <%= link_to_with_icon 'check', Spree.t(:confirm), confirm_admin_order_url(@order) %>
       </li>
     <% end %>
 
     <% if can? :display, Spree::ReturnAuthorization %>
       <% if @order.completed? %>
-        <li<%== ' class="active"' if current == 'Return Authorizations' %> data-hook='admin_order_tabs_return_authorizations'>
-          <%= link_to_with_icon 'share', Spree::ReturnAuthorization.model_name.human(count: :other), admin_order_return_authorizations_url(@order) %>
+        <li class="tab <%= "active" if current == "Return Authorizations" %>" data-hook='admin_order_tabs_return_authorizations'>
+          <%= link_to_with_icon 'share', Spree.t(:rma), admin_order_return_authorizations_url(@order) %>
```

Comment:
We currently do a bunch of work to use `model_name.human` everywhere. So introducing a new `rma` key is not preferred here. So we should change the translation of the mode name here. Although this will have wider consequences. 

Using the `admin.tab` scope would be ok for me.